src/jit/emitxarch.cpp

```diff
@@ -1925,7 +1925,7 @@ UNATIVE_OFFSET      emitter::emitInsSizeAM(instrDesc* id, size_t code)
         // Most 16-bit operands will require a size prefix .
         // This refers to 66h size prefix override.
 
-        if  (    (attrSize == EA_2BYTE)
+        if  (    attrSize == EA_2BYTE
```

Comment:
This is unfortunate.  Our coding conventions require that this be parenthesized, as in the FEATURE_STACK_FP_X87 case there are further subexpressions.  If you really want to change this, you will need to change the #if so that it encloses the entire if statement, so that it will conform to the coding conventions.